feature-group-definitions/array-flat-and-flatmap.yml

```diff
@@ -0,0 +1,6 @@
+caniuse: array-flat
```

Comment:
Any reason to not match caniuse here and call this feature `array-flat`? This also has implications for https://github.com/web-platform-dx/feature-set/pull/120 if the answer is yes.

Reply:
`array-flat` is now consistent with `array-group`. See https://github.com/web-platform-dx/feature-set/pull/120#discussion_r1168897684.